api/src/test/java/org/openmrs/ConceptTest.java

```diff
@@ -1101,4 +1101,40 @@ public void getDescriptions_shouldNotReturnNullIfDescriptionsListIsNull() throws
 		Assert.assertNotNull(c.getDescriptions());
 	}
 
+	/**
+	 * @see Concept#hasName(String, Locale)
+	 * @verifies hasName returns false if name parameter Is Null
+	 */
+	@Test
+	public void hasName_shouldReturnFalseIfNameIsNull()
+	{
+		Concept concept = new Concept();
+		concept.addName(new ConceptName("Test Concept", new Locale("en"))) ;
+		Locale localeToSearch = new Locale("en", "UK");
+		Assert.assertFalse(concept.hasName(null, localeToSearch));
+	}
+
+	/**
+	 * @see Concept#hasName(String, Locale)
+	 * @verifies hasName returns ture if locale parameter Is Null but name is found
+	 */
+	@Test
+	public void hasName_shouldReturnTrueIfLocaleIsNullButNameIsFoundInLocale()
+	{
```

Comment:
Does each of these tests have corresponding should annotations on the method being tested?